tools/clang_format.py

```diff
@@ -28,8 +28,9 @@ def get_clang_format_config_with_columnlimit(rootdir, limit):
         text=True,
     )
     proc.check_returncode()
+
     return re.sub(
-        r"(ColumnLimit:\s*)\d+", r"\g<1>{}".format(BREAK_BEFORE), proc.stdout
+        r"(ColumnLimit:\s*)\d+", r"\g<1>{}".format(limit), proc.stdout
```

Comment:
\+ this minor fix, the function didn't use its `limit` argument before and used the constant directly (though with the same effect)